frontend/src/app/components/wp-table/embedded/wp-embedded-table-entry.component.ts

```diff
@@ -1,8 +1,9 @@
 import {Component, ElementRef, Input, OnInit} from '@angular/core';
-import {DynamicBootstrapper} from "core-app/globals/dynamic-bootstrapper";
+
+export const wpTableEntrySelector = 'wp-embedded-table-entry';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/3b1eae5a64159f8a044579a25870c772186dbd3f/frontend/src/app/components/wp-table/embedded/wp-embedded-table-entry.component.ts#L4"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>